### ISSUE TITLE
Year of publication of the method

### ISSUE THREAD

**Issue by jpainam** [natowi/3D-Reconstruction-with-Neural-Network#3]
If you could add the years of publication. Putting recent methods first

**Comment by natowi** [comment]
I agree, this will be useful. I´ll add it with the next update. There have been many new publications since my last update
